File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        int count = 0;
        int maxCnt = Integer.MIN_VALUE;

        for(int i = 0 ; i < nums.length; i++){
            if(nums[i] == 1){
                count++;
            }
            if(nums[i] == 0){
                maxCnt =  Math.max(count,maxCnt);
                count = 0;
            }
        }
        
        maxCnt = Math.max(count, maxCnt);
        return maxCnt == Integer.MIN_VALUE ? 0: maxCnt;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        max_ones = 0
        ones = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                ones = ones+1
            if nums[i] == 0:
                max_ones = max(max_ones,ones)
                ones = 0
        
        max_ones = max(max_ones,ones)
        return max_ones

        